src/styles/utils/_vars.scss

```diff
@@ -1 +1,18 @@
-$c-gray: #eee;
+$c-bg-color: #f7f7f7;
+$c-main-dark-color: #131313;
+$c-white-color: #fff;
+$c-btn-default-color: #333;
+$c-btn-hover-color: #131313;
+$c-disabled-color: #828282;
+$c-main-text-color: #7c7c7c;
+$c-main-link-color: #0db2b3;
+$c-secondary-text-light-color: #bdbdbd;
+$c-input-color: #828282;
+$c-send-btn-hover-color: #0c797a;
+$c-invalid-input-color: #eb5757;
+$c-secondary-bg-color: #d6ecec;
```

Comment:
Do not abbreviate names to unreadable ones, when expanding your code, abbreviations can be confusing.